src/pretix/plugins/badges/exporters.py

```diff
@@ -167,7 +165,8 @@ def _renderer(event, layout):
 
 
 def render_pdf(event, positions, opt):
-    from PyPDF2 import PdfReader, PdfWriter
+    from PyPDF2 import PdfMerger, PdfReader, PdfWriter, Transformation
+    from PyPDF2.generic import RectangleObject
```

Comment:
I think we can import this all at top level? Not sure why it was done like this before.

Reply:
Yeah, me neither, but thought it was on purpose. I will change accordingly.